tests/perception/perception_graph_test.py

```diff
@@ -919,6 +920,21 @@ def test_simulated_one_object_graph():
         return False
 
 
+def test_rgb_color_node():
+    assert RgbColorNode(red=0, blue=0, green=0, weight=1.0)
+    assert RgbColorNode(red=255, blue=255, green=255, weight=0.0)
+
+
+def test_rgb_color_predicate():
+    assert RgbColorPredicate(red=0, blue=0, green=0, weight=1.0)
+    assert RgbColorPredicate(red=255, blue=255, green=255, weight=0.0)
+
+
+def test_rgb_color_perception():
+    assert RgbColorPerception(red=0, blue=0, green=0)
+    assert RgbColorPerception(red=255, blue=255, green=255)
+
+
```

Comment:
Minor changes:

1. Since these are boundary tests, put that in the name? `test_rgb_color_{name}_accepts_valid_values` say
2. No need to assert the objects -- `attrs` will raise an error when validation fails leading to test failure. The real thing we're testing is whether these raise an error, so you can just call the constructors without the assert. If the pre-commit linters complain, disable them for these lines.a

Reply:
Thanks for the tips! I've updated the tests accordingly.